Parse-Dashboard/index.js

```diff
@@ -105,8 +107,8 @@ p.then(config => {
 
   const app = express();
 
-  if (allowInsecureHTTP) app.enable('trust proxy');
-  app.use(mountPath, parseDashboard(config.data, allowInsecureHTTP));
+  if (allowInsecureHTTP || trustProxy) app.enable('trust proxy');
```

Comment:
Can you have the dashboard throw an error if both `allowInsecureHTTP` and `trustProxy` are set? We want to nudge people away from using `allowInsecureHTTP`.